documentation/modules/retrieving-vmware-mor.adoc

```diff
@@ -0,0 +1,81 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+:_content-type: PROCEDURE
+[id="retrieving-vmware-mor_{context}"]
+= Retrieving a VMware vSphere MOR
+
+When you migrate VMs with a VMware vSphere source provider using {project-first} from the CLI, you need to know the managed object reference (MOR) of certain entities in vSphere, such as datastores, networks, and VMs.
+
+You can retrieve the MOR of one or more vSphere datastores from the Inventory service. You can then use each MOR as a reference for retrieving the MOR of another entity.
```

Comment:
I'm not sure about this part - it applies to snapshots since we start from the datastore that the corresponding disks reside on, but if we take virtual machines - they are not placed on a datastore so if we need their MOR, the datastore won't be their entrypoint

I'd suggest to generalize the first sentence to be more general, e.g., `You can retrieve the MOR of one or more vSphere entities from the Inventory service.`

Reply:
Fixed.